app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>User details</h1>
\ No newline at end of file
+<h1>User details</h1>
+
+<div>
+  <div class="ml-3">
+  <h4 class="border ps-2 w-25 m-2 p-4 border-dark"> <%= @user.name %> </h4> 
+  </div>
+     <div class="border h-50 m-2 w-50 p-2 border-dark">
+      <span> bio</span> <br>
+      <%= @user.bio %>
+
+      </div><br>
+
+      <% @user.recent_posts.each do |post| %>
+       <div class="border h-50 w-50 p-2 m-2 border-dark">
+        <%= post.title %> <br> <br>
+        <%= post.text %>
+       </div>
+      <% end %>
+
+      <img src="<%= @user.photo %>"><br>
+        <%# <%= render partial: 'shared/user-card', locals: { user: user } %>
```

Comment:
- [ ] Please consider removing all pieces of commented code as they make your code base less readable! 🤓 🔧